apps/src/templates/studioHomepages/TeacherSections.jsx

```diff
@@ -59,23 +50,6 @@ function TeacherSections({
           />
         </ContentContainer>
       )}
-      {shouldRenderPlSections() && (
-        <ContentContainer heading={i18n.plSectionsTitle()}>
-          <BodyTwoText>
-            {i18n.myProfessionalLearningSectionsHomepageDesc()}
```

Comment:
I'm wondering if we should remove the i18n strings alongside the references. I started a slack thread [here](https://codedotorg.slack.com/archives/C045UAX4WKH/p1718723307149209).

Reply:
Answer from Mario in the thread is that if there are no references to the string it should be removed